tests/test_diffeq/test_odefiltsmooth/test_odefiltsmooth.py

```diff
@@ -320,17 +312,13 @@ def test_error_ibm1(self):
     def test_error_ibm2(self):
         """Expect error rate q+1 """
         stp1, stp2 = self.stps
-        ms, cs, ts = probsolve_ivp(
-            self.ivp, step=stp1, method="eks0", which_prior="ibm2"
-        )
-        means1 = ms[:, 0]
-        sols1 = np.array([self.ivp.solution(t) for t in ts])
+        sol = probsolve_ivp(self.ivp, step=stp1, method="eks0", which_prior="ibm2")
+        means1 = sol.y.mean()
+        sols1 = np.array([self.ivp.solution(t) for t in sol.t])
         err1 = np.amax(np.abs(sols1[:, 0] - means1))
```

Comment:
Add an assert: len(means1.shape) == 1 to make sure that the tests will indeed fail when the Univariate stuff gets fixed.